.github/workflows/ci.yaml

```diff
@@ -116,21 +107,22 @@ jobs:
           name: ruff
           path: target/debug/ruff
 
-  cargo-fuzz:
```

Comment:
Git really struggling with the diff here. Just view the diffs of https://github.com/astral-sh/ruff/pull/8287/commits/ae3aa2032f247c1f20fd9b663a65212c30dac7d4 and https://github.com/astral-sh/ruff/pull/8287/commits/9735b7f98311d3521736dcdfdbaeedb1ec914168 separately. The later commit just swaps the fuzz/wasm test jobs.